### DISCUSSION TITLE
citofono analogico urmet

### DISCUSSION THREAD

**Discussion by barbatrukko** [indomus/forum#2048] [HUB: Home Assistant]
Salve, vorrei riuscire ad effettuare 2 controlli su questa tipologia di citofono e non so se si può fare ed eventualmente come.
Si tratta del classico citofono urmet analogico, vorrei riuscire a fare questo:

1. disattivarlo in una certa fascia oraria, basta anche solo "interrompere" il filo che alimenta il buzzer.
2. aprire il citofono da HomeAssistant (e quindi con qualsiasi tipo di automazione o pulsante). Quindi simulare la pressione del tasto di apertura.

In pratica in un caso vorrei staccare la suoneria e nell'altro simulare la pressione del tasto di apertura.
E' possibile? si può fare qualcosa con dei sonoff mini ad esempio? o con cosa?
Grazie

**Comment by Scialla** [comment]
ti serve un interruttore a contatto pulito (es shelly1) per l'apertura della porta, ma lo shelly o similare lo devi alimentare esternamente, non puoi dal citofono. Ti basta collegarlo in parallelo al pulsante del citofono.

Stessa cosa per la suoneria, se trovi il filo del buzzer

**Reply by barbatrukko** [reply]
si, la seccatura del citofono è che è alimentato a 12AC.
Quindi prendo shelly1, lo alimento per fatti miei, collego i pin 0 e 1 in parallelo al pulsante del citofono (magari da software faccio prolungare l'impulso per 1 sec).
Stessa cosa per il buzzer: collego il buzzer in serie al contatto 0 1 e funzionerà solo quando da app chiudo il contatto?

**Reply by Scialla** [reply]
Si, per apertura in parallelo al pulsante del citofono, un secondo glielo dai nelle impostazioni dello Shelly, per il buzzer devi individuare il filo e interromperlo con altro Shelly (non lo stesso) e lo usi come interruttore

**Reply by barbatrukko** [reply]
Si, si sono 2 citofoni diversi e quindi 2 shelly1. Li ho ordinati, appena arrivano provo.
Per ora grazie.

**Reply by Larry-78** [reply]
Ciao barbatrukko il citofono in questione è Urmet 1130? Di solito è utilizzato per impianti a 5 fili, se confermi allora il 6 e 9 sono i contatti che dovresti cortocircuitare per l'apertura e il 7 è di solito il contatto da interrompere per evitare la chiamata, ad ogni modo ogni modifica sull'impianto se non effettuata correttamente potrebbe creare disturbi o nella peggiore delle ipotesi, malfunzionamenti alla montante citofonica...i condomini saranno felicissimi.

**Comment by barbatrukko** [comment]
Buonasera, ho effettuato l'intervento sul primo citofono, per consentire di aprire il portone da remoto ed ha funzionato.
Nei prossimi giorni mi occuperò del secondo citofono, quello del quale voglio disattivare in certi orari il cicalino.
